### PR TITLE
Update Connect wallet

### PR DESCRIPTION
**Pull request by foopod**

### CODE REVIEW

File: src/pages/Mint/Mint.tsx [hunk 1]
```diff
@@ -11,7 +12,12 @@ const Profile: React.FC = () => {
 
 	if (!web3Provider) {
 		// Require wallet connection
-		return <ConnectWallet />
+		return (
+			<>
+				<Header />
```
Comment (ScreamingHawk) [inline src/pages/Mint/Mint.tsx:17]: Remove `hasHeader{false}` instead

File: src/pages/ConnectWallet/ConnectWallet.tsx [hunk 2]
```diff
@@ -1,14 +1,45 @@
...
+					>
+						{MINT_HERO_BUTTON_TEXT}
+					</Button>
+					<p>
+						<sub>{MINT_HERO_SUBTEXT}</sub>
+					</p>
+					<img className={classes.barfHand} src="/img/barf-hand.png" />
+				</section>
+				<JaggySection color={white} direction={'up'} />
+			</div>
```
Comment (ScreamingHawk) [inline src/pages/ConnectWallet/ConnectWallet.tsx:38]: Idk if this should be part of the connect wallet section

File: src/pages/ConnectWallet/ConnectWallet.tsx [hunk 3]
```diff
@@ -1,14 +1,45 @@
...
+			<div className={classes.section}>
 				<ConnectButton />
 			</div>
+			<Footer />
```
Comment (ScreamingHawk) [inline src/pages/ConnectWallet/ConnectWallet.tsx:42]: Remove `hasFooter={false}` instead

**Review by ScreamingHawk — CHANGES_REQUESTED** [review]
I'm going to ignore this PR for now and focus on the after wallet connect functionality. Definitely needs more work before it's in a usable state

File: src/pages/ConnectWallet/ConnectWallet.tsx (hunk 2 above)
Comment (ScreamingHawk) [reply src/pages/ConnectWallet/ConnectWallet.tsx:38]: Maybe keep it but replace the button with the `ConnectWallet` button

Comment (ScreamingHawk) [reply src/pages/ConnectWallet/ConnectWallet.tsx:38]: Would be nice if this primary section is it's own shared component across both `Home` and here

**Comment by ScreamingHawk** [comment]
After doing #7 I think we can drop this. The design is a bit different to the wireframe. What do you think?

**Comment by foopod** [comment]
Closing as not merging, see above comments.

**Pull request closed without merging**